openlibrary/coverstore/utils.py

```diff
@@ -11,21 +8,18 @@
 import random
 import string
 
-import config
-import oldb
+from six.moves.urllib.parse import splitquery, unquote, unquote_plus
+from six.moves.urllib.parse import urlencode as real_urlencode
```

Comment:
Ahhh, that's annoying :P Looks like the "fake" urlencode is dead code (only used in some silenced tests), but let's deal with it later. This seems like a good interim solution :+1: